### ISSUE TITLE
Parse Revision documents

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/pubid-ieee#13]
Some PubIDs show the document it revised. In this case we need to parse the new PubID and the PubID of the revised document, separately. When we generate the PubID, it should show the two new PubIDs.

```
AIEE No.4-1942 (Revision of AIEE No.4-1940)
IEEE Std 1491-2012 (Revision of IEEE Std 1491-2005)
IEEE Std C95.2-2018 (Revision of IEEE Std C95.2-1999)
```

**Comment by mico** [comment]
@ronaldtse "ANSI C37.53.1-1989 (R1996) (Revision of ANSI C37.53.1-1982)" (https://ieeexplore.ieee.org/document/1389208)
I'm wondering what is (R1996) here. Looks like a revision and year, but it's not 1996 year...

**Comment by ronaldtse** [comment]
Yes I agree this looks like Reaffirm. Maybe this is the short form "Ryyyy". Let's use this as the short form and "Reaffirmed yyyy" as the long form?

**Comment by mico** [comment]
> Yes I agree this looks like Reaffirm. Maybe this is the short form "Ryyyy". Let's use this as the short form and "Reaffirmed yyyy" as the long form?

But we don't have short form for Revision / Amendment / etc, or we have?

**Comment by ronaldtse** [comment]
We don't have short forms for others... unless you can find them! :-)
